app/components/common_post_options/follow_thread_option/index.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+
+import {observeCurrentTeamId} from '@app/queries/servers/system';
+
+import FollowThreadOption from './follow_thread_option';
+
+import type {WithDatabaseArgs} from '@typings/database/database';
+
+const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
+    return {
+        teamId: observeCurrentTeamId(database),
```

Comment:
I could open the post options for a post in a permalink view that does not necessarily belong to the current team.